webextensions/manifest/optional_permissions.json

```
@@ -26,7 +26,7 @@
           "__compat": {
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "21"
```

```suggestion
                "version_added": "26"
```

Was introduced in 26, not 21. See the main review comment attached to this review for a detailed write-up on how to correctly identify the version.